documentai/snippets/handle_response_sample_test.py

```diff
@@ -64,7 +64,7 @@ def test_process_document_form(capsys):
     expected_strings = [
         "There are 1 page(s) in this document.",
         "Table with 4 columns and 6 rows",
-        "Found 13 form field(s)",
+        "form field(s)",
         "'BALANCE DUE': '$2140.00'",
```

Comment:
In Python 3.7 this fails as the output is differently formatted:
```
'BALANCE DUE' | '$2140.00'
```

Maybe it's time to consider redoing the test to check returned data instead of the output?

Reply:
Probably so. I left this one using capsys because these samples are about handling the response, rather than the request itself. But I changed the function to check returns instead of prints.

Reply:
And tests are passing now

Reply:
Python guidance has changed recently (to return values). Thank you for your flexibility, @holtskinner .